data/styles/vehicle/include/Icons.mapcss

```diff
@@ -242,6 +242,7 @@ node|z16-[amenity=community_centre],
 area|z16-[amenity=community_centre],
 node|z16-[historic=boundary_stone],
 area|z16-[historic=boundary_stone],
+node|z16-[boundary=marker],
```

Comment:
for the vehicle (navigation) style I think its not needed at all, it'll be a distraction for drivers

Reply:
I agree, removed